Gems/EditorPythonBindings/Code/Source/PythonLogSymbolsComponent.h

```diff
@@ -51,12 +51,19 @@ namespace EditorPythonBindings
 
         ////////////////////////////////////////////////////////////////////////
         // PythonSymbolEventBus::Handler
-        void LogClass(AZStd::string_view moduleName, AZ::BehaviorClass* behaviorClass) override;
-        void LogClassWithName(AZStd::string_view moduleName, AZ::BehaviorClass* behaviorClass, AZStd::string_view className) override;
-        void LogClassMethod(AZStd::string_view moduleName, AZStd::string_view globalMethodName, AZ::BehaviorClass* behaviorClass, AZ::BehaviorMethod* behaviorMethod) override;
-        void LogBus(AZStd::string_view moduleName, AZStd::string_view busName, AZ::BehaviorEBus* behaviorEBus) override;
-        void LogGlobalMethod(AZStd::string_view moduleName, AZStd::string_view methodName, AZ::BehaviorMethod* behaviorMethod) override;
-        void LogGlobalProperty(AZStd::string_view moduleName, AZStd::string_view propertyName, AZ::BehaviorProperty* behaviorProperty) override;
+        void LogClass(const AZStd::string moduleName, const AZ::BehaviorClass* behaviorClass) override;
+        void LogClassWithName(const AZStd::string moduleName, const AZ::BehaviorClass* behaviorClass, const AZStd::string className) override;
+        void LogClassMethod(
+            const AZStd::string moduleName,
+            const AZStd::string globalMethodName,
+            const AZ::BehaviorClass* behaviorClass,
+            const AZ::BehaviorMethod* behaviorMethod) override;
+        void LogBus(const AZStd::string moduleName, const AZStd::string busName, const AZ::BehaviorEBus* behaviorEBus) override;
+        void LogGlobalMethod(const AZStd::string moduleName, const AZStd::string methodName, const AZ::BehaviorMethod* behaviorMethod) override;
+        void LogGlobalProperty(
+            const AZStd::string moduleName,
+            const AZStd::string propertyName,
+            const AZ::BehaviorProperty* behaviorProperty) override;
```

Comment:
All the `const AZStd::string` parameters should be `AZStd::string` to allow movement of the strings into the queued function container